Source/ACE.Entity/Character.cs

```diff
@@ -173,6 +177,12 @@ private Character()
 
             InitializeCharacterOptions();
             CharacterOptions = new ReadOnlyDictionary<CharacterOption, bool>(characterOptions);
+
+            // initialize the blank character positions
+            InitializeCharacterPositions();
+            CharacterPositions = new ReadOnlyDictionary<CharacterPositionType, CharacterPosition>(characterPositions);
+            PhysicalPosition = new Position(CharacterPositions[CharacterPositionType.PhysicalLocation].cell, CharacterPositions[CharacterPositionType.PhysicalLocation].positionX, CharacterPositions[CharacterPositionType.PhysicalLocation].positionY, CharacterPositions[CharacterPositionType.PhysicalLocation].positionZ, CharacterPositions[CharacterPositionType.PhysicalLocation].rotationX, CharacterPositions[CharacterPositionType.PhysicalLocation].rotationY, CharacterPositions[CharacterPositionType.PhysicalLocation].rotationZ, CharacterPositions[CharacterPositionType.PhysicalLocation].rotationW);
```

Comment:
seems like it would be helpful here to have a Position.Clone() method so that you didn't have to do this inline.